queries.md

```diff
@@ -0,0 +1,51 @@
+# Query documentation
+
+## Highlights
+
+### Description of query
+
+The highlights query is composed of two parts:
+
+( A ) Select **1 item** from History where:
```

Comment:
I would say "attempt to select" for the first spot. If there is no such bookmark, the first spot will be replaced by items in the second part of the query